auto-generated/vector-crypto/policy_funcs/llvm-overloaded-tests/vaesdf.c

```diff
@@ -0,0 +1,47 @@
+// REQUIRES: riscv-registered-target
+// RUN: %clang_cc1 -triple riscv64 -target-feature +v -disable-O0-optnone \
+// RUN:   -emit-llvm %s -o - | opt -S -passes=mem2reg | \
+// RUN:   FileCheck --check-prefix=CHECK-RV64 %s
+
+#include <riscv_vector.h>
+
+vuint32mf2_t test_vaesdf_vv_u32mf2_tu(vuint32mf2_t vd, vuint32mf2_t vs2, size_t vl) {
+  return __riscv_vaesdf_tu(vd, vs2, vl);
```

Comment:
I think it should have both `_vv` and `_vs` version since it should be able to determine what instruction to use in backend.